pages/docs/suspense.mdx

```diff
@@ -0,0 +1,44 @@
+import Callout from 'components/callout'
+
+# Suspense
+
+<Callout emoji="🚨" background="bg-red-200">
+  Suspense is current an <strong>experimental</strong> feature of React. These features may change significantly and without a warning before they become a part of React. 
+  <br/>
+  <a href="https://reactjs.org/docs/concurrent-mode-suspense.html" target="_blank">More information</a>
+</Callout>
+
+<Callout>
+  Note that React Suspense is not yet supported in SSR mode.
+</Callout>
+
+You can enable the `suspense` option to use SWR with React Suspense:
+
+```jsx
+import { Suspense } from 'react'
+import useSWR from 'swr'
+
+function Profile () {
+  const { data } = useSWR('/api/user', fetcher, { suspense: true })
+  return <div>hello, {data.name}</div>
+}
+
+function App () {
+  return (
+    <Suspense fallback={<div>loading...</div>}>
+      <Profile/>
+    </Suspense>
+  )
+}
+```
+
+In Suspense mode, `data` is always the fetch response (so you don't need to check if it's `undefined`). 
+But if an error occurred, you need to use an [error boundary](https://reactjs.org/docs/concurrent-mode-suspense.html#handling-errors) to catch it:
+
```

Comment:
it would be grate if we could have more details for conditional key + suspense mode here.
like you commented here before `https://github.com/vercel/swr/pull/357#issuecomment-627089889`

maybe another section of alerting callout message here? 👆

Reply:
Good idea!